+ source /root/mugen/libs/locallibs/common_lib.sh
++ python3 --version
++ '[' 0 -eq 0 ']'
++ source /root/mugen/libs/locallibs/common_lib_python.sh
+ main
++ type -t post_test
+ '[' -n function ']'
+ trap post_test EXIT INT HUP TERM
+ rpm -qa
+ grep expect
++ type -t config_params
+ '[' -n '' ']'
++ type -t pre_test
+ '[' -n '' ']'
++ type -t run_test
+ '[' -n function ']'
+ run_test
+ LOG_INFO 'Start to run test.'
+ message='Start to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to run test.'
Thu Jul 27 19:09:06 2023 - INFO  - Start to run test.
+ SSH_CMD reboot
+ cmd=reboot
+ remoteip=
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u root -p 'openEuler12#$' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 19:09:08 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ REMOTE_REBOOT_WAIT 2 15
+ node=2
+ waittime=15
+ '[' 15x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 15
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ SSH_CMD reboot
+ cmd=reboot
+ remoteip=
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u root -p 'openEuler12#$' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 19:09:15 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ REMOTE_REBOOT_WAIT 2 15
+ node=2
+ waittime=15
+ '[' 15x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 15
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ SSH_CMD reboot
+ cmd=reboot
+ remoteip=
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c reboot -i '' -u root -p 'openEuler12#$' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 19:09:23 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ REMOTE_REBOOT_WAIT 2 15
+ node=2
+ waittime=15
+ '[' 15x '!=' x ']'
+ python3 /root/mugen/libs/locallibs/remote_reboot.py wait --node 2 --waittime 15
Traceback (most recent call last):
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 126, in <module>
    sys.exit(reboot_wait(args.node, args.waittime))
  File "/root/mugen/libs/locallibs/remote_reboot.py", line 62, in reboot_wait
    "ping -c 3 -w 3 " + os.environ.get("NODE" + str(node) + "_IPV4")
TypeError: can only concatenate str (not "NoneType") to str
+ SSH_CMD 'last reboot > /tmp/rebootlog2'
+ cmd='last reboot > /tmp/rebootlog2'
+ remoteip=
+ remotepasswd='openEuler12#$'
+ remoteuser=root
+ timeout=300
+ connport=22
+ bash /root/mugen/libs/locallibs/sshcmd.sh -c 'last reboot > /tmp/rebootlog2' -i '' -u root -p 'openEuler12#$' -t 300 -o 22
Usage: sshcmd.sh -c "command" -i "remote machinet ip" [-u login_user] [-p login_password] [-o port] [-t timeout]+ ret=1
+ test 1 -ne 0
+ LOG_ERROR 'Failed in remote CMD operation:1'
+ message='Failed in remote CMD operation:1'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote CMD operation:1'
Thu Jul 27 19:09:30 2023 - ERROR - Failed in remote CMD operation:1
+ return 1
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_system_monitor_reboot.sh line 30'
+ message='oe_test_system_monitor_reboot.sh line 30'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_system_monitor_reboot.sh line 30'
Thu Jul 27 19:09:31 2023 - ERROR - oe_test_system_monitor_reboot.sh line 30
+ '[' 0 -eq 1 ']'
+ return 0
+ SSH_SCP @:/tmp/rebootlog2 .
+ src=@:/tmp/rebootlog2
+ dest=.
+ remotepasswd='openEuler12#$'
+ connport=22
+ bash /root/mugen/libs/locallibs/sshscp.sh -p 'openEuler12#$' -o 22 -s @:/tmp/rebootlog2 -d .
Thu Jul 27 19:09:33 2023 - WARN  - the remote password uses the default configuration.
Thu Jul 27 19:09:35 2023 - WARN  - the connect port using the default configuration
spawn scp -P 22 -r @:/tmp/rebootlog2 .
ssh: Could not resolve hostname : Name or service not known
scp: Connection closed
+ ret=255
+ test 255 -ne 0
+ LOG_ERROR 'Failed in remote SCP operation: 255'
+ message='Failed in remote SCP operation: 255'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'Failed in remote SCP operation: 255'
Thu Jul 27 19:09:37 2023 - ERROR - Failed in remote SCP operation: 255
+ return 255
+ CHECK_RESULT 255
+ actual_result=255
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 255 ']'
+ '[' 0 -eq 0 ']'
+ test 255x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_system_monitor_reboot.sh line 32'
+ message='oe_test_system_monitor_reboot.sh line 32'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_system_monitor_reboot.sh line 32'
Thu Jul 27 19:09:39 2023 - ERROR - oe_test_system_monitor_reboot.sh line 32
+ '[' 0 -eq 1 ']'
+ return 0
++ cat rebootlog2
++ grep reboot
++ wc -l
cat: rebootlog2: No such file or directory
+ num_user=0
+ test 0 -ge 3
+ CHECK_RESULT 1
+ actual_result=1
+ expect_result=0
+ mode=0
+ error_log=
+ exit_mode=0
+ '[' -z 1 ']'
+ '[' 0 -eq 0 ']'
+ test 1x '!=' 0x
+ test -n ''
+ (( exec_result++ ))
+ LOG_ERROR 'oe_test_system_monitor_reboot.sh line 35'
+ message='oe_test_system_monitor_reboot.sh line 35'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'oe_test_system_monitor_reboot.sh line 35'
Thu Jul 27 19:09:40 2023 - ERROR - oe_test_system_monitor_reboot.sh line 35
+ '[' 0 -eq 1 ']'
+ return 0
+ LOG_INFO 'End to run test.'
+ message='End to run test.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to run test.'
Thu Jul 27 19:09:42 2023 - INFO  - End to run test.
+ CASE_RESULT 0
+ case_re=0
+ test -z 3
+ test 3 -gt 0
+ LOG_ERROR 'failed to execute the case.'
+ message='failed to execute the case.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level error --message 'failed to execute the case.'
Thu Jul 27 19:09:43 2023 - ERROR - failed to execute the case.
+ exit 3
+ post_test
+ LOG_INFO 'Start to restore the test environment.'
+ message='Start to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'Start to restore the test environment.'
Thu Jul 27 19:09:44 2023 - INFO  - Start to restore the test environment.
+ rm -rf rebootlog2
+ LOG_INFO 'End to restore the test environment.'
+ message='End to restore the test environment.'
+ python3 /root/mugen/libs/locallibs/mugen_log.py --level info --message 'End to restore the test environment.'
Thu Jul 27 19:09:46 2023 - INFO  - End to restore the test environment.
